### ISSUE TITLE
Implement mutex lock for the `SendAPIRequest()` method to avoid multiprocessing issues

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#92]
https://superfastpython.com/multiprocessing-in-python/#Process_Mutex_Lock
https://docs.python.org/3/library/multiprocessing.html
https://superfastpython.com/threadpool-join/

**Comment by Tim55667757** [comment]
```
from multiprocessing import cpu_count, Lock
...
self.__lock = Lock()  # initialize multiprocessing mutex lock
...
with self.__lock:  # acquire the mutex lock
...
```

**Comment by Tim55667757** [comment]
Implemented since v1.6.dev121 builds

Debug build: https://app.travis-ci.com/github/Tim55667757/TKSBrokerAPI/builds/258170093
Debug PyPI artefact: https://pypi.org/project/tksbrokerapi/1.6.dev121/
